MapboxNavigation/MapboxVoiceController.swift

```diff
@@ -257,7 +257,7 @@ open class MapboxVoiceController: RouteVoiceController, AVAudioPlayerDelegate {
 
 //MARK: - Obsolete
 extension MapboxVoiceController {
-    @available(*, deprecated, renamed: "MapboxVoiceController.play(instruction:data:)")
+    @available(*, deprecated, message: "It is now required that the maneuver's `SpokenInstruction` is passed into `play` when calling. Please use MapboxVoiceController.play(instruction:data:).")
```

Comment:
Similarly, should we obsolete this method too? v1.0 is our last chance before v2.0 to remove anything, so do we want to start it out with older cruft? It is harsh to outright remove a method without first deprecating it, but v1.0 is already a harsh migration; we can mitigate these removals with a migration guide.